doc/src/installation.rst

```diff
@@ -0,0 +1,279 @@
+manually or in login scripts, to fix the Cylc version in their environment.
+
+Installing subsequent releases is just a matter of unpacking the new tarballs
+next to the previous releases, running ``make`` in them, and copying
+in (possibly with modifications) the previous site global config file.
+
+
+.. _LocalInstall:
+
+Local User Installation
+^^^^^^^^^^^^^^^^^^^^^^^
+
+It is easy to install Cylc under your own user account if you don't have
+root or sudo access to the system: just put the central Cylc wrapper in
+``$HOME/bin/`` (making sure that is in your ``$PATH``) and
+modify it to point to a directory such as ``$HOME/cylc/`` where you
+will unpack and install release tarballs. Local installation of third party
+dependencies like Graphviz is also possible, but that depends on the particular
+installation methods used and is outside of the scope of this document.
+
+Create A Site Config File
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Site and user global config files define some important parameters that affect
+all suites, some of which may need to be customized for your site.
+See :ref:`SiteAndUserConfiguration` for how to generate an initial site file and
+where to install it. All legal site and user global config items are defined
+in :ref:`SiteRCReference`.
+
+
+.. _Configure Site Environment on Job Hosts:
+
+Configure Site Environment on Job Hosts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your users submit task jobs to hosts other than the hosts they use to run
+their suites, you should ensure that the job hosts have the correct environment
+for running cylc. A cylc suite generates task job scripts that normally invoke
+``bash -l``, i.e. it will invoke bash as a login shell to run the job
+script. Users and sites should ensure that their bash login profiles are able
+to set up the correct environment for running cylc and their task jobs.
+
+Your site administrator may customise the environment for all task jobs by
+adding a ``<cylc-dir>/etc/job-init-env.sh`` file and populate it with the
+appropriate contents. If customisation is still required, you can add your own
+``${HOME}/.cylc/job-init-env.sh`` file and populate it with the
+appropriate contents.
+
+- ``${HOME}/.cylc/job-init-env.sh``
+- ``<cylc-dir>/etc/job-init-env.sh``
+
+The job will attempt to source the first of these files it finds to set up its
+environment.
+
+
+.. _ConfiguringCylcReviewApache:
+
+Configuring Cylc Review Under Apache
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Cylc Review web service displays suite job logs and other information in
+web pages - see :ref:`ViewingSuiteLogsCylcReview` and
+:numref:`fig-review-screenshot`. It can run under a WSGI server (e.g.
+Apache with ``mod_wsgi``) as a service for all users, or as an ad hoc
+service under your own user account.
+
+To run Cylc Review under Apache, install ``mod_wsgi`` and configure it
+as follows, with paths modified appropriately:
+
+.. code-block:: bash
```

Comment:
```suggestion
.. code-block:: apacheconf
```

Reply:
Fancy! The Pygments lexer must be pretty comprehensive. Thanks for noticing this.